spec/helpers-spec.js

```diff
@@ -0,0 +1,39 @@
+'use babel';
+
+import * as path from 'path';
+// eslint-disable-next-line no-unused-vars, import/no-extraneous-dependencies
+import { it, fit, wait, beforeEach, afterEach } from 'jasmine-fix';
+import * as helpers from '../lib/helpers';
```

Comment:
Since `helpers.js` is using Module syntax instead of CommonJS I believe you can write this as `import helpers from '../lib/helpers';`.

Reply:
I think because of the way ```helpers.js``` uses the Module syntax it has to be written like this. When I write it as ```import helpers from '../lib/helpers';``` eslint gives this error: ```Caution: `helpers` also has a named export `findStylelintDirectory`. Check if you meant to write `import {findStylelintDirectory} from '../lib/helpers'` instead. (import/no-named-as-default-member)```

Reply:
Ah, nevermind then!